schema.sql

```diff
@@ -1,10 +1,9 @@
 /* Database schema to keep the structure of entire database. */
-
-vet_clinic=# CREATE TABLE animals(
-id integer generated by default as identity
-(start with 1 increment by 1),
+vet_clinic = # CREATE TABLE animals(
```

Comment:
Please be sure that you use 

`ALTER TABLE animals ADD species varchar(255);` 

in order to add species column into your animal table 🤓

Reply:
Added now, thank you!
<img width="892" alt="Screen Shot 2022-07-19 at 21 02 16" src="https://user-images.githubusercontent.com/25887253/179819364-f1ea3f20-8b63-4b55-b0a7-5406124ab399.png">